.github/workflows/build.yml

```diff
@@ -11,18 +11,17 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: azure/docker-login@v1
+      - uses: actions/checkout@v4
+      - uses: azure/docker-login@v2
         with:
           login-server: quay.io
           username: '${{ secrets.QUAYIO_USERNAME }}'
           password: '${{ secrets.QUAYIO_PASSWORD }}'
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
The action by `crazy-max` is archived, they recommend the QEMU + builds action combination as replacement in their README.